homeassistant/components/yolink/binary_sensor.py

```diff
@@ -28,17 +34,35 @@ class YoLinkBinarySensorEntityDescription(BinarySensorEntityDescription):
     value: Callable[[str], bool | None] = lambda _: None
 
 
-SENSOR_DEVICE_TYPE = [ATTR_DEVICE_DOOR_SENSOR]
+SENSOR_DEVICE_TYPE = [
+    ATTR_DEVICE_DOOR_SENSOR,
+    ATTR_DEVICE_MOTION_SENSOR,
+    ATTR_DEVICE_LEAK_SENSOR,
+]
 
 SENSOR_TYPES: tuple[YoLinkBinarySensorEntityDescription, ...] = (
     YoLinkBinarySensorEntityDescription(
         key="state",
         icon="mdi:door",
         device_class=BinarySensorDeviceClass.DOOR,
-        name="state",
+        name="State",
         value=lambda value: value == "open",
         exists_fn=lambda device: device.device_type in [ATTR_DEVICE_DOOR_SENSOR],
     ),
+    YoLinkBinarySensorEntityDescription(
+        key="state",
+        device_class=BinarySensorDeviceClass.MOTION,
+        name="Motion",
+        value=lambda value: value == "alert",
+        exists_fn=lambda device: device.device_type in [ATTR_DEVICE_MOTION_SENSOR],
+    ),
+    YoLinkBinarySensorEntityDescription(
+        key="state",
+        name="Leak",
```

Comment:
```suggestion
                device_class=BinarySensorDeviceClass.MOISTURE,
                name="Leak",
```
This is the closest one to indicate leak